lute/templates/bookmarks/list.html

```diff
@@ -0,0 +1,198 @@
+{% extends 'base.html' %}
+
+{% block title %}Bookmarks{% endblock %}
+{% block header %}Bookmarks{% endblock %}
+
+{% block body %}
+
+<h1>{{ book.title }}</h1>
+
+<table id="bookmarkstable" class="table dataTable stripe">
+  <thead>
+    <tr>
+      <th style="text-align: left">Page</th>
+      <th style="text-align: left">Description</th>
+      <th style="text-align: left">Actions</th>
+    </tr>
+  </thead>
+</table>
+
+<!-- the config widget for datatables.  Moved into the datatables table footer during setup. -->
```

Comment:
I don't think we need this config widget but it doesn't hurt I guess!  Very minor point.

Reply:
I'd prefer to keep it tbh. Could be useful when I add more stuff to the bookmarks 🙂

Reply:
Sounds good!